src/main/java/plugily/projects/buildbattle/handlers/themes/ThemeManager.java

```diff
@@ -40,56 +44,133 @@
+    themesConfig.set("Themes.Classic", classicThemes);
+    themesConfig.set("Themes.Teams", teamsThemes);
+    themesConfig.set("Themes.Guess-The-Build.Easy", GTBThemesEasy);
+    themesConfig.set("Themes.Guess-The-Build.Medium", GTBThemesMedium);
+    themesConfig.set("Themes.Guess-The-Build.Hard", GTBThemesHard);
+
+    ConfigUtils.saveConfig(plugin, themesConfig, "themes");
+  }
+
+  public void loadThemes(boolean ignoreHolidays) {
+    // Remove all colours from theme names
+    classicThemes.replaceAll(ChatColor::stripColor);
+    teamsThemes.replaceAll(ChatColor::stripColor);
+    GTBThemesEasy.replaceAll(ChatColor::stripColor);
+    GTBThemesMedium.replaceAll(ChatColor::stripColor);
+    GTBThemesHard.replaceAll(ChatColor::stripColor);
+
+    if(!ignoreHolidays && !plugin.getHolidayManager().getEnabledHolidays().isEmpty()) {
       loadSpecialThemes();
       return;
     }
-    gameThemes.put(BaseArena.ArenaType.SOLO.getPrefix(), themes.getStringList("Themes.Classic"));
-    gameThemes.put(BaseArena.ArenaType.TEAM.getPrefix(), themes.getStringList("Themes.Teams"));
-    gameThemes.put(BaseArena.ArenaType.GUESS_THE_BUILD.getPrefix() + "_EASY", themes.getStringList("Themes.Guess-The-Build.Easy"));
-    gameThemes.put(BaseArena.ArenaType.GUESS_THE_BUILD.getPrefix() + "_MEDIUM", themes.getStringList("Themes.Guess-The-Build.Medium"));
-    gameThemes.put(BaseArena.ArenaType.GUESS_THE_BUILD.getPrefix() + "_HARD", themes.getStringList("Themes.Guess-The-Build.Hard"));
+
+    gameThemes.put(GameThemes.SOLO, Collections.unmodifiableList(classicThemes));
+    gameThemes.put(GameThemes.TEAM, Collections.unmodifiableList(teamsThemes));
+    gameThemes.put(GameThemes.GUESS_THE_BUILD_EASY, Collections.unmodifiableList(GTBThemesEasy));
+    gameThemes.put(GameThemes.GUESS_THE_BUILD_MEDIUM, Collections.unmodifiableList(GTBThemesMedium));
+    gameThemes.put(GameThemes.GUESS_THE_BUILD_HARD, Collections.unmodifiableList(GTBThemesHard));
   }
 
   private void loadSpecialThemes() {
     for(Holiday holiday : plugin.getHolidayManager().getEnabledHolidays()) {
-      List<String> themeList = themes.getStringList("Holiday." + holiday.getName());
+      List<String> themeList = themesConfig.getStringList("Holiday." + holiday.getName());
+
       if(themeList.isEmpty()) {
         continue;
       }
-      themeList.addAll(gameThemes.getOrDefault("Classic", new ArrayList<>()));
-      gameThemes.put("Classic", themeList);
-      gameThemes.put("Teams", themeList);
-      gameThemes.put("Guess-The-Build", themeList);
-    }
-  }
 
-  public Map<String, List<String>> getGameThemes() {
-    return gameThemes;
+      Optional.ofNullable(gameThemes.get(GameThemes.CLASSIC)).ifPresent(themeList::addAll);
+
+      gameThemes.put(GameThemes.CLASSIC, Collections.unmodifiableList(themeList));
+      gameThemes.put(GameThemes.TEAMS, Collections.unmodifiableList(themeList));
+      gameThemes.put(GameThemes.GUESS_THE_BUILD, Collections.unmodifiableList(themeList));
```

Comment:
that one was wrong on old code already, gtb does only have easy medium and hard, so it would need to be added to all of that types we are ignoring the difficulty in this case as special stuff can be in every